3.0/deploy/docker/prometheus/prometheus.yml

```diff
@@ -0,0 +1,27 @@
+# my global config
+global:
+  scrape_interval:     3s # Set the scrape interval to every 3 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+
+# Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+rule_files:
+# - "first.rules"
+# - "second.rules"
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: 'prometheus'
+    # metrics_path defaults to '/metrics'
+    # scheme defaults to 'http'.
+    static_configs:
+      - targets: ['localhost:9090']
+
+  - job_name: 'dubbo-go-benchmark'
+    # metrics_path defaults to '/metrics'
+    # scheme defaults to 'http'.
+    static_configs:
+      - targets: ['your-computer-ip:9091']
```

Comment:
Why is read ip used here instead of localhost? I think you could use host network in `docker-compose.yml`.

Reply:
If it must use the real ip, then, please rename this file to `prometheus.yml.example` and add `prometheus.yml` to `.gitignore` to  avoid synchronizing the yaml file with git.

Reply:
> The host networking driver only works on Linux hosts, and is not supported on Docker for Mac, Docker for Windows, or Docker EE for Windows Server.

My computer is Windows and there is no way to use the host network or verify that the host network is configured correctly, so for the time being I will use the real ip of host.